scipy/signal/windows/_windows.py

```diff
@@ -1213,6 +1214,81 @@ def kaiser(M, beta, sym=True):
     return _truncate(w, needs_trunc)
 
 
+def kaiser_bessel_derived(M, beta, *, sym=True):
+    """Return a Kaiser-Bessel derived window.
+
+    Parameters
+    ----------
+    M : int
+        Number of points in the output window. If zero or less, an empty
+        array is returned. Note that this window is only defined for an even
+        number of points. It will raise `ValueError` otherwise.
+    beta : float
+        Kaiser window shape parameter.
+    sym : bool, optional
+        This parameter only exists to comply with the interface offered by
+        the other window functions and to be callable by `get_window`.
+        When True (default), generates a symmetric window, for use in filter
+        design. It will raise `ValueError` otherwise.
+
+    Returns
+    -------
+    w : ndarray
+        The window, normalized to fulfil the Princen-Bradley condition.
+
+    Notes
+    -----
+    It is designed to be suitable for use with the modified discrete cosine
+    transform (MDCT) and is mainly used in audio signal processing and
+    audio coding.
+
+    .. versionadded:: 1.9.0
+
+    References
+    ----------
+    .. [1] Wikipedia, "Kaiser window",
+           https://en.wikipedia.org/wiki/Kaiser_window
+
+    Examples
+    --------
+    Plot the Kaiser-Bessel derived window based on the wikipedia
+    reference [1]_:
+
+    >>> from scipy import signal
+    >>> import matplotlib.pyplot as plt
+    >>> fig, ax = plt.subplots()
+    >>> N = 100
+    >>> for alpha in [0.64, 2.55, 7.64, 31.83]:
+    ...     ax.plot(signal.windows.kaiser_bessel_derived(N, alpha),
+    ...             label=f"{alpha=}")
+    >>> ax.grid(True)
+    >>> ax.set_title("Kaiser-Bessel derived window")
+    >>> ax.set_ylabel("Amplitude")
+    >>> ax.set_xlabel("Sample")
+    >>> fig.legend(loc="center")
+    >>> fig.tight_layout()
+    >>> fig.show()
```

Comment:
We are passing `alpha` with the same values as in the wiki. But the function is parametrized using `beta` so it should be `np.pi * alpha`.

Looking at the wiki, I would also use the same axis (add 0.707 on y-axis and n, 2N-1 on x-axis)

Reply:
Indeed. I improved the example figure. What do you think?
<img width="573" alt="SS 2022-01-05 at 13 29 49" src="https://user-images.githubusercontent.com/3813847/148160816-8badece8-1919-4a21-bc91-2b745c5e05d0.png">

Reply:
This looks great thanks!